src/ape/api/query.py

```diff
@@ -149,8 +189,8 @@ def estimate_query(self, query: QueryType) -> Optional[int]:
             query (``QueryType``): Query to estimate.
 
         Returns:
-            Optional[int]: Represents milliseconds, returns ``None`` if unable to execute.
-
+            Optional[``QueryEstimateType``]: Represents an estimate to perform the query,
+                                            returns ``None`` if unable to execute.
```

Comment:
Nit, but otherwise it adds some unnecessary indentation in the rendering:

```suggestion
            returns ``None`` if unable to execute.
```